dotcom-rendering/scripts/gen-stories/get-stories.mjs

```diff
@@ -202,7 +281,7 @@ ${storyName}.decorators = [splitTheme([{
 
 /** @typedef {{display: string, design: string, theme: string, config: import('../../src/types/configContext.js').Config}} TestFormat */
```

Comment:
Not a blocker, but it feels like there’s a bit of loose coupling now between the `renderingTarget` field in `TestFormat` and the `testLayoutWebFormats` and `testLayoutAppsFormats`. Maybe we can simplify that to be automatically in both cases?